cmd/network/network.go

```diff
@@ -25,7 +25,17 @@ func init() {
 	NetworkCmd.AddCommand(networkCreateCmd)
 	NetworkCmd.AddCommand(networkUpdateCmd)
 	NetworkCmd.AddCommand(networkRemoveCmd)
+	NetworkCmd.AddCommand(networkShowCmd)
+	NetworkCmd.AddCommand(networkConnectCmd)
 
 	networkCreateCmd.Flags().StringVarP(&cidrV4, "cidr-v4", "", "", "Custom IPv4 CIDR")
 	networkCreateCmd.Flags().StringSliceVarP(&nameserversV4, "nameservers-v4", "", nil, "Custom list of IPv4 nameservers (up to three, comma-separated)")
+
+	// Flag binding for networkConnectCmd
+	networkConnectCmd.Flags().IntVar(&vlanID, "vlan-id", 0, "VLAN ID to connect")
+	networkConnectCmd.Flags().StringVar(&vlanCIDRV4, "cidr-v4", "", "CIDR v4 of the VLAN")
+	networkConnectCmd.Flags().StringVar(&vlanGatewayIPv4, "gateway-ip-v4", "", "Gateway IPv4 address for the VLAN")
```

Comment:
can you change it into `gateway-ipv4` so it's the same as the JSON `gateway_ipv4`

Reply:
https://github.com/civo/cli/pull/379/commits/b9676dc5ba8794dc277c1cc1d5e7383141b3bbad